addons/web/views/webclient_templates.xml

```diff
@@ -167,6 +167,8 @@
 
         <script type="text/javascript" src="/web/static/src/js/tools/debug_manager.js"></script>
 
+        <script type="text/javascript" src="/web/static/src/js/common_env.js"></script>
```

Comment:
Should also be added to assets_frontend

Reply:
? here we are in assets_common

Reply:
You can ignore this comment then, my bad